setup.py

```diff
@@ -56,11 +56,4 @@ def get_version():
         'Topic :: Software Development',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    entry_points = {
```

Comment:
This entry_points are mandatory otherwise sqlalchemy isn't able to find the dialect implementation for HANA DBURIs.

Reply:
as I understood from README.dialects one either Needs to put entry_points in Setup.py or registry.Register() in conftest.py, so i thought it was redundant to have both of them.